core/object/script_language.h

```diff
@@ -256,6 +256,7 @@ class ScriptLanguage : public Object {
 	virtual bool can_inherit_from_file() const { return false; }
 	virtual int find_function(const String &p_function, const String &p_code) const = 0;
 	virtual String make_function(const String &p_class, const String &p_name, const PackedStringArray &p_args) const = 0;
+	virtual bool can_make_function() { return true; }
```

Comment:
I'd suggest adding a virtual method for this to `ScriptLanguageExtension` so we don't forget about it

Reply:
Double-checking before I do something wrong. We're talking about adding an `EXBIND`, and a `GDVIRTUAL_BIND` in `ScriptLanguageExtension`, right?

Reply:
Yep! There should be good examples of `bool` returns there that can be copied